ee/ucp/kubernetes/storage/use-azure-disk.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Configuring Azure Disk Storage for Kubernetes
+description: Learn how to add persistent storage to your Docker Enterprise clusters running on Azure with Azure Disk. 
+keywords: Universal Control Plane, UCP, Docker EE, Kubernetes, storage, volume
+redirect_from:
+---
+
+Operators can provide persistent storage for workloads running on Docker
+Enterprise when running on Microsoft Azure by using Azure Disk. Operators can
+either pre-provision Azure Disks to be consumed by Kubernetes Pods, or can use
+the Azure Kubernetes integration to dynamically provision Azure Disks on demand.
+
+
+## Pre-Requisites
```

Comment:
Remove `-` to make [prerequisites](https://www.merriam-webster.com/dictionary/prerequisite)